### ISSUE TITLE
Last error in WEBUI, when no error happened

### ISSUE THREAD

**Issue by EmilioMoreno** [distribworks/dkron#1612]
**Describe the bug**

Web Interface "Last Error" showing error timestamp, but job execution list shows no error
We have a job every 15 minutes which failed several times about 3 days ago.  The last error field and the specific job execution list are a match.

Today without any error on the job execution list,  the  "Last Error" field on the Jobs section at the WEBUI has been updated, several times during the day, but always the execution list is clear. No failures shown, no retries, nothing, all the executions for that job are showing a check.

**To Reproduce**
Sorry, no clear steps to reproduce

**Expected behavior**
If the job section show a Last Error timestamp, the corresponding execution in the execution list should be in error at that date

**Screenshots**
![CleanShot 2024-11-12 at 20 51 23](https://github.com/user-attachments/assets/9cb78f57-a292-435c-923e-6ecc249d0cd1)
![CleanShot 2024-11-12 at 20 52 05](https://github.com/user-attachments/assets/22f5c8c1-1cb6-4d4c-a2f8-289d5e7eb053)

** Specifications:**
Agents and servers running linux, 3.2.7
5 servers, 4 agents available for this specific job. 

**Additional context**

**Comment by vcastellm** [comment]
This is pretty strange, I couldn't reproduce something like this, did you edit the job around that time?

**Comment by vcastellm** [comment]
Hi @EmilioMoreno can you check with latest version?

**Comment by EmilioMoreno** [comment]
@vcastellm i assume you mean  latest 4 version. Is it ready for production?

**Comment by vcastellm** [comment]
@EmilioMoreno yes, it's ready for production but there's a known bug with the shell executor, non critical and we're working on it.

**Comment by EmilioMoreno** [comment]
@vcastellm i am affected by #1483 too, so in production i rather wait for the next version.  However, I will update my development environment to 4.0.2 and let you know if it happens again or not. 

To your question:  " did you edit the job around that time?" i am not sure; all our jobs are updated on every deployment so it might have been the case.

**Comment by vcastellm** [comment]
@EmilioMoreno #1483 is fixed and we're going to release the next version.
